sycl/doc/extensions/supported/sycl_ext_oneapi_sub_group_mask.asciidoc

```diff
@@ -100,6 +113,37 @@ The mask is defined such that the least significant bit (LSB) corresponds to
 the work-item with id 0, and the most significant bit (MSB) corresponds to the
 work-item with the id `max_local_range()-1`.
 
+|===
+|Constructor|Description
+
+|`sub_group_mask()`
+|Constructs a group mask with all bits set to 0. Size of a group mask
+corresponds to max local range of a sub-group which work-item belongs to.
+
+|`sub_group_mask(unsigned long long val)`
+|Constructs a group mask with the first `N` bit positions to the
+corresponding bit values in _val_. `N` is a size of a group mask and it
+corresponds to max local range of a sub-group which work-item belongs to. If
+size of a group mask is bigger than number of bits in the value representation
```

Comment:
```suggestion
size of a group mask is bigger than the number of bits in the value representation
```

Reply:
Applied in 366c3ea5f81dbaec0d4429f593f6e74b72eec266